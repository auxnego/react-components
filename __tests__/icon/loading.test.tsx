import React from "react";
import {create} from "react-test-renderer";

import LoadingIcon from "../../icon/loading";

it("simple loading icon", () => {

  expect(create(
    <LoadingIcon />
  ).toJSON()).toMatchSnapshot();

});
