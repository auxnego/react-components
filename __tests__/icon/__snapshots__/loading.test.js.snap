// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple loading icon 1`] = `
<i
  className="fa fa-spinner animate-spin"
/>
`;
